app/assets/javascripts/appear.js

```diff
@@ -5,9 +5,10 @@ $(function(){  // $(document).ready shorthand
 
 $(document).ready(function() {
     
+      "asgbaoiaawgaweg awgawgeaiwogjaweg awegoiawjegaw egaoiwjgaw egoaiwejga wgeoiajweg aweogijweg "  
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Trailing whitespace.<br>Missing semicolon.